test/legacy_test/test_zero_dim_no_backward_api.py

```diff
@@ -487,37 +487,39 @@ def test_one_hot_label(self):
         self.assertEqual(res[0].shape, (4,))
         self.assertEqual(res[0][2], 1)
 
+    @test_with_pir_api
     def test_unique_consecutive(self):
         x = paddle.rand([])
         y, inverse, counts = paddle.unique_consecutive(
             x, return_inverse=True, return_counts=True
         )
 
         prog = paddle.static.default_main_program()
-        res = self.exe.run(prog, fetch_list=[y, inverse, counts])
-        self.assertEqual(y, x)
-        self.assertEqual(inverse, 0)
-        self.assertEqual(counts, 1)
-        self.assertEqual(res[0].shape, (1,))
+        res = self.exe.run(prog, fetch_list=[x, y, inverse, counts])
```

Comment:
```suggestion
        x_res, y_res, inverse_res, counts_res = self.exe.run(prog, fetch_list=[x, y, inverse, counts])
```

这样一目了然，不然下面取下标还得看 0、1、2、3 分别是啥